<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>information</class>
 <widget class="QDialog" name="information">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>896</width>
    <height>688</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>查看/修改信息</string>
  </property>
  <property name="whatsThis">
   <string/>
  </property>
  <widget class="QLabel" name="lbTitle">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>10</y>
     <width>871</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cambria Math</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>课 程 表</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="schedule">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>630</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>课程表</string>
   </property>
  </widget>
  <widget class="QPushButton" name="students">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>630</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>学生</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btnexit">
   <property name="geometry">
    <rect>
     <x>772</x>
     <y>630</y>
     <width>111</width>
     <height>28</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>退出(不保存)</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>88</y>
     <width>877</width>
     <height>50</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>50</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="6">
     <widget class="QTextEdit" name="textEdit_2"/>
    </item>
    <item row="0" column="8">
     <widget class="QPushButton" name="pushButton">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="BtnSearch">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>查找</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QTextEdit" name="textEdit"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>查找：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="7">
     <widget class="QPushButton" name="BtnAdd">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>添加</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QPushButton" name="BtnCancel">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>添加：</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="homework">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>630</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>作业</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnEdit">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>630</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>确认修改</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>861</width>
     <height>441</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnDrop">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>630</y>
     <width>91</width>
     <height>28</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>删除选中</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
